api/tools/src/main/java/org/pmiops/workbench/tools/CreateTerraMethodSnapshot.java

```diff
@@ -65,6 +68,32 @@
 
   private static final Logger log = Logger.getLogger(CreateTerraMethodSnapshot.class.getName());
 
+  private static final Pattern importWdlRegex = Pattern.compile("import \"(.+)\" as (.+)");
```

Comment:
Is the alias a required part of the import statement in WDL?

Reply:
ah, its not. I made some changes to account for that